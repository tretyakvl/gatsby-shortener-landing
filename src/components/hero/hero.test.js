import React from 'react'
import renderer from 'react-test-renderer'
import Hero from './hero'

describe('Hero', () => {
  it('renders correctly', () => {
    const props = {
      title: 'Very cool service',
      desc: 'What an amazing description!',
      cta: {
        to: '/buy-the-product-please',
        text: 'No refunds'
      }
    }

    const tree = renderer
      .create(<Hero {...props} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
