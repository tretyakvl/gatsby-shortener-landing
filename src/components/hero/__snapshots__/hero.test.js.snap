// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero renders correctly 1`] = `
<section
  className="hero"
  style={
    Object {
      "opacity": 0,
      "transform": "translateX(-20px) translateZ(0)",
    }
  }
>
  <img
    alt="Illustration of a woman working on a computer"
    height="482"
    src="test-file-stub"
    width="733"
  />
  <div
    className="hero__container"
  >
    <h2
      className="hero__header"
    >
      Very cool service
    </h2>
    <p
      className="hero__text"
    >
      What an amazing description!
    </p>
    <a
      className="button-module button"
      href="/buy-the-product-please"
    >
      No refunds
    </a>
  </div>
</section>
`;
